docker-images/kafka-based/kafka/scripts/kafka_run.sh

```diff
@@ -55,6 +51,26 @@ echo ""
 # Configure heap based on the available resources if needed
 . ./dynamic_resources.sh
 
+# Prepare for Kraft
+if [ -n "$STRIMZI_CLUSTER_ID" ]; then
```

Comment:
@tombentley Cluster ID indeed exists even without KRaft. But only with KRaft we pass it to the brokers. In other cases, it is loaded from ZooKeeper (or from the storage?). So it can be reliably used to determine the KRaft mode.

Do you want me to add a a separate environment variable to enable the KRaft mode? It doe snot seem necessary. But it is easy to do. So I can add it if you want.

Reply:
Done.

Reply:
Thanks!